src/effects/effectrack.cpp

```diff
@@ -105,8 +127,11 @@ EffectChainSlotPointer EffectRack::getEffectChainSlot(int i) {
     return m_effectChainSlots[i];
 }
 
+// NOTE(Kshitij) : Why are we using a clone of the EffectChain?
```

Comment:
This concept of cloning chains does not make sense with the new architecture. Comment out this function for now and add a TODO comment noting to come back to it when we implement loading chain presets from the user's list of saved presets. Likewise for EffectRack::loadPrevChain.

Reply:
Actually these functions don't belong in EffectRack. They should be functions of EffectChainSlot. There's no need for the EffectRack layer and it should be removed.